Automattic/wp-calypso
# Tests: Move Search tests to single test runner

This is part of #3942 Makefile migration. Running the test on the mocha runner worked without having to make and changes. I also verified that it will pass with `.only` on the outer `describe`. 
I can note that on the issue with √  after merging.

```
$ npm run test-client -- --grep "components search"

> wp-calypso@0.17.0 test-client /Users/jason/Projects/wp-calypso
> NODE_ENV=test NODE_PATH=test:client test/runner.js "--grep" "components search"



  components
    search
      Search
        initialValue
          with initialValue
            ✓ should set state.keyword with the initialValue after mount
          without initialValue
            ✓ should set state.keyword empty string after mount


  2 passing (270ms)

```

**Testing Instructions**

verify all tests with `npm run test-client`
verify isolation test by adding `.only` to any of the `describe` blocks and run `npm run test-client`

cc @gziolo
